serving/docker/partition/partition.py

```diff
@@ -181,14 +181,20 @@ def run_partition(self) -> str:
         logging.info(f"cmd: {commands}")
         self.set_environmental_vars()
         partition_stdout = ""
+        partition_stderr = ""
         # Use Popen to capture stdout without delaying terminal output
         with subprocess.Popen(commands,
                               stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE,
                               bufsize=1,
-                              universal_newlines=True) as proc:
+                              text=True) as proc:
             for line in proc.stdout:
                 partition_stdout += line
                 print(line, end='')
+            # Exception is the last line of stderr
+            for line in proc.stderr:
+                pass
+            partition_stderr = line
```

Comment:
if `stderr` is empty, this will accidentally capture the last line of `stdout`.

Reply:
Fixed:
https://github.com/deepjavalibrary/djl-serving/blob/2f131026b30d1c4ea72641b395e2a1b0b5341b14/serving/docker/partition/partition.py#L195-L196